backend-project/small_eod/generic/validators.py

```diff
@@ -0,0 +1,17 @@
+from django.core.validators import BaseValidator
+from django.utils.translation import ugettext_lazy as _
+
+
+class ExactLenghtsValidator(BaseValidator):
+    message = _(
+        "Ensure this value has length of any %(limit_value)s (it has %(show_value)d)."
+    )
+
+    def __init__(self, limit_value: list = None, **kwargs):
+        super().__init__(limit_value=limit_value, **kwargs)
+
+    def compare(self, a, b):
+        return all([a != x for x in b])
```

Comment:
```suggestion
        return all((a != x for x in b))
```

Reply:
Jaka jest roznica?

Reply:
Dobra praktyka, aby używać generatorów, gdy jest to możliwe. Generatory sprawiają, że nie wszystkie dane są wczytywane do pamięci, a tylko tyle ile jest konieczne w danym momencie.

Reply:
pyupgrade wyrzuca tutaj blad ale nie ma informacji jaki